src/plugins/spaces_oss/public/types.ts

```diff
@@ -8,21 +8,41 @@
 
 import type { SpacesApi } from './api';
 
+/**
+ * OSS Spaces Plugin start contract when the spaces feature is enabled.
+ */
 export interface SpacesAvailableStartContract extends SpacesApi {
+  /** Indicates if the spaces feature is enabled. */
   isSpacesAvailable: true;
 }
 
+/**
+ * OSS Spaces plugin start contract when the spaces feature is disabled.
+ * @deprecated The spaces plugin will always be enabled starting in 8.0.
```

Comment:
I know that we typically do not capitalize feature names in our docs, but I also know that Spaces is sometimes an explicit exception to that rule. @gchaps can you help us out?

My guess is that we would prefer `plugin` over `Plugin`.

Reply:
I recommend "Spaces plugin" because it is the name of a plugin.  This makes it different than a "space", which is where users organize their saved objects.

Reply:
> I recommend "Spaces plugin" because it is the name of a plugin. This makes it different than a "space", which is where users organize their saved objects.

So, the same logic would apply to "Spaces feature", right?

Reply:
Yes